main.js

```diff
@@ -175,7 +176,7 @@ exports.main = function (options, callbacks) {
   });
 
   hotkey = Hotkey({
-    combo: 'accel-shift-l',
+    combo: strings.toggleHotkey,
```

Comment:
error in the console:
<img width="890" alt="screen shot 2016-11-08 at 2 41 52 pm" src="https://cloud.githubusercontent.com/assets/10803178/20114285/a0589d42-a5c1-11e6-99d8-1b2ce664a3f8.png">
<img width="881" alt="screen shot 2016-11-08 at 2 41 34 pm" src="https://cloud.githubusercontent.com/assets/10803178/20114293/a4c70ff8-a5c1-11e6-895d-302c23fb135e.png">